server/middlewares/verifyToken.js

```diff
@@ -10,12 +10,12 @@ export default (request, response, next) => {
   const token = request.headers.authorization || request.query.token;
 
   if (token) {
-    jwt.verify(token, process.env.JWT_KEY, async (error, decoded) => {
+    jwt.verify(token, process.env.JWT_KEY, async(error, decoded) => {
```

Comment:
Do you have eslint installed as a VSCode extension? It should flag it that you should add a space between the async keyword and the open bracket.